busybox_config

```diff
@@ -792,7 +792,7 @@ CONFIG_FEATURE_LESS_MAXLINES=0
 #
 # Networking Utilities
 #
-# CONFIG_FEATURE_IPV6 is not set
+CONFIG_FEATURE_IPV6=y
```

Comment:
not sure - maybe you should provide also custom busybox config only for ia32? (as for `armv7a9-zynq7000-zedboard`)